ArchiSteamFarm/Bot.cs

```diff
@@ -3912,6 +3920,154 @@ internal sealed class Bot : IDisposable {
 			return responses.Count > 0 ? string.Join("", responses) : null;
 		}
 
+		private static async Task<string> ResponseTransfer(ulong steamID) {
```

Comment:
Once above is done, remove this entirely, there is no need to define such wrapper.